dashboard/app/components/ui/sidebar/app.sidebar.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
```

Comment:
remove, this is not a concept in React router

Reply:
fixed